src/modules/Home/components/MainSlider/MainSlider.tsx

```diff
@@ -0,0 +1,168 @@
+import cn from 'classnames';
+import React, {PureComponent} from 'react';
+
+import Arrow from 'assets/images/ImageArrowDown';
+import Flex from 'common/components/Flex/Flex';
+import MainSlide from '../../containers/MainSlideContainer';
+import ProgressCircle from './components/ProgressCircle/ProgressCircle';
+
+import styles from './mainSlider.scss';
+
+interface ISlide {
+	order: number;
+	content: any;
+	name: 'string';
```

Comment:
what does it meens? type? value?

Reply:
Fixed.
By the way, there is no such word "meens".